packages/react-next/src/components/Slider/__snapshots__/Slider.test.tsx.snap

```diff
@@ -33,24 +22,18 @@ exports[`Slider renders correctly 1`] = `
           margin-right: 0px;
           margin-top: 0px;
           overflow-wrap: break-word;
-          padding-bottom: 0px;
-          padding-left: 0px;
-          padding-right: 0px;
-          padding-top: 0px;
+          padding-bottom: 5px;
+          padding-left: 0;
+          padding-right: 0;
+          padding-top: 5px;
```

Comment:
Interesting that we are seeing a change here in the actual attributes as opposed to just the classname being applied.

Reply:
I spent a fair amount of time checking into it, I think that this makes sense since the underlying component is ultimately a Label, so we'd expect to see a mix of css and css-in-js. I think at least.

Reply:
So all of those inline styles are the css-in-js being applied from the underlying label.